index.html

```diff
@@ -4,9 +4,180 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title class="title">Quiz App!</title>
+    <title>Quiz App!</title>
+    <link rel="stylesheet" href="styles.css" />
   </head>
-  <body>
-    <h1>Quiz App</h1>
+
+  <body class="structure">
+    <main>
+      <h1 class="structure__header">Quiz</h1>
+
+      <section class="structure__card">
```

Comment:
maybe it would make more sense to give your structure_class class the name "card" as the block element.